prober/prober.sh

```diff
@@ -0,0 +1,11 @@
+set -eEuo pipefail
+
+ID_TOKEN=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_token
+
+cat jvs_token | ./jvsctl validate --token -
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_breakglass_token
+
+cat jvs_breakglass_token | ./jvsctl validate --token -
```

Comment:
Breakglass is not necessary because it won't contact server if it's breakglass.

Reply:
Is it useful to add breakglass here so that we know whether the breakglass CUJ works or not?